multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/SyncFlowableStep.java

```diff
@@ -188,7 +191,8 @@ protected ProcessStepHelper getStepHelper() {
             stepHelper = new ProcessStepHelper(getProgressMessageService(),
```

Comment:
This class already has 4 parameters, which is a bit much, and changing it to 5 isn't recommended.

Reply:
Used immutable instead